src/assets/styles/main.scss

```diff
@@ -0,0 +1,6 @@
+@import 'reset.css';
+@import 'global.scss';
+
+/*
+** Main
```

Comment:
Does this file even need to exist anymore?

Reply:
Yes, main is for shared styling across all components